components/packages/pb-input/src/pb-input.ts

```diff
@@ -0,0 +1,190 @@
+  private placeholder: string = "";
+
+  @property({ attribute: 'value', type: String, reflect: true })
+  private value: string = "";
+
+  @property({ attribute: 'type', type: String, reflect: true })
+  private type: string = "text";
+
+  @property({ attribute: 'min', type: String, reflect: true })
+  private min: string = "";
+
+  @property({ attribute: 'max', type: String, reflect: true })
+  private max: string = "";
+
+  @property({ attribute: 'step', type: String, reflect: true })
+  private step: string = "1";
+
+
+  constructor() {
+    super();
+    listenForLangChanged(() => {
+      // TODO: enable translation
+      // We do not use translate() directive in render() since placeholder is an exposed attribute
+      // this.placeholder = get("placeholder");
+    });
+  }
+
+  async attributeChangedCallback(name: string, old: string|null, value: string|null) {
+    super.attributeChangedCallback(name, old, value);
+    if (name === 'lang') {
+      use(this.lang).then();
+    }
+  }
+
+  static getCatalog(lang: string) {
+    switch(lang) {
+      case "es":
+      case "es-ES":
+        return i18n_es;
+      case "fr":
+        return i18n_fr;
+      case "ja":
+        return i18n_ja;
+      case "pt":
+      case "pt-BR":
+        return i18n_pt;
+      default:
+        return i18n_en;
+    }
+  }
+
+  static get styles() {
+    return css`
+      :host {
+        display: block;
+        font-family: sans-serif;
+        text-align: left;
+        padding: 10px 10px 10px 0px;
+        max-width: var(--max-width, 50%);
+      }
+
+      .input-elem {
+        font-size: 14px;
+        height: 20px;
+      }
+      
+      .label-elem {
+        font-size: 14px;
+        font-weight: 700;
+        margin-bottom: 5px;
+      }
+      
+      /* Add a red star after required inputs */
+      .label-required:after {
+        content: "*";
+        color: #C00;
+      }
+
+    `;
+  }
+
+  render() {
+    return html`
+      <style>${bootstrapStyle}</style>
+      ${this.getLabel()}
+      <input 
+        class="${this.getInputCssClass()}"
+        id="input"
+        name="${this.name}"
+        type="${this.type}"
+        min="${this.min}"
+        max="${this.max}"
+        step="${this.step}"
+        value="${this.value}"
+        @input=${(e: any) => this.valueChanged(e.target.value)} 
+        placeholder="${this.placeholder}"
+        minlength="${this.minLength}"
+        maxlength="${this.maxLength}"
+        ?readonly="${this.readOnly}"
+      />
+    `;
+  }
+
+  private getLabel() {
+    if (this.labelHidden) {
+      return html``;
+    }
+    return html`
+        <label
+          class="${this.getLabelCssClass()}"
+          for="input"
+        >${this.label}</label>
+        `
+  }
+
+  private getLabelCssClass() : string {
+    return "control-label label-elem " + (this.required ? "label-required" : "") +
+      " col-xs-" + (!this.labelHidden && this.labelPosition === 'left' ? this.labelWidth : 12);
```

Comment:
I don't think we want to manage only col-xs size... What about md, lg or others ?

Reply:
I did was was done in the legacy pbInput template. To be discussed if needed.